_studio/mfx_lib/encode_hw/hevc/linux/base/hevcehw_base_va_lin.cpp

```diff
@@ -176,6 +176,15 @@ void DDI_VA::InitAlloc(const FeatureBlocks& /*blocks*/, TPushIA Push)
 
         m_callVa = Glob::DDI_Execute::Get(strg);
 
+        //Get max priority
+        std::vector<VAConfigAttrib> attrib_priority(1);
+        attrib_priority[0].type = VAConfigAttribContextPriority;
+        sts = MfxEncodeHW::DeviceVAAPI::QueryCaps(attrib_priority.data(), attrib_priority.size() * sizeof(VAConfigAttrib));
+        MFX_CHECK_STS(sts);
+        PriorityParam priorityPar;
+        priorityPar.m_MaxContextPriority = attrib_priority[0].value;
```

Comment:
same as above - check whether attribute is supported.

Reply:
Done.